File: PRAKTIKUM 4/PRAK405_2310817220028_SheilaSabina.c
#include <stdio.h>

int main() {
    int s, e, l, a, angka = 0;

    printf("Input\n");
    scanf("%d %d", &s, &e);

    for (l = 1; l <= s; l++) {
        int row_angka = 0;
        for (a = 1; a <= l; a++) {
            int result = a * e;
            printf("(%d * %d) = %d", a, e, result);
            if (a != l) {
                printf(" + ");
            }
            row_angka += result;
        }
        printf(" = %d\n", row_angka);
        angka += row_angka;
    }

    printf("%d", angka);

    return 0;
}

File: PRAKTIKUM 4/PRAK405_2310817220028_SheilaSabina.py
s, e = map(int, input("Input\n").split())

angka = 0

for l in range(1, s + 1):
    row_angka = 0
    for a in range(1, l + 1):
        result = a * e
        print(f"({a} * {e}) = {result}", end="")
        if a != l:
            print(" + ", end="")
        row_angka += result
    print(f" = {row_angka}")
    angka += row_angka

print(angka)